### DISCUSSION TITLE
npc_vendor npc_trainer 9.0.2 (37176)

### DISCUSSION THREAD

**Discussion by rem999** [TrinityCore/TrinityCore#25968] [Q&A] [answered]
npc_vendor npc_trainer Where are the settings? In the client or in the kernel? Previously, it was possible to make changes through the database. Can't you return the old days? Or with this now, strictly?

npc_vendor npc_trainer Где находятся настройки? В клиенте или в ядре? Раньше можно было через базу данных вносить изменения. Нельзя вернуть былое время? Или с этим сейчас, строго?

**Comment by Shauren** [comment, marked as answer]
Speak only english.

`npc_vendor` is still there
`npc_trainer` is not coming back, it was replaced with `trainer`, `trainer_spell`, `creature_trainer`

**Comment by rem999** [comment]
Да работает. Спасибо человек! С доброй душой. 
Yes it works. Thanks man! With a kind soul.
